app/helpers/spotlight/browse_helper.rb

```diff
@@ -24,6 +24,12 @@ def default_document_index_view_type
       end
     end
 
+    ##
+    # Get parent results count of a browse category search
+    def parent_search_count
+      Spotlight::Search.find(@search.id).count
```

Comment:
I get a bit nervous about this.  This triggers a solr request under the hood to get the document count.  Having this in a helper like this could trigger multiple page blocking solr requests w/o realizing it.

A few options I can think of initially.
1. Memoize this helper method
1. Move this logic into the `Spotlight::Search` class and memoize it there (could get into a weird state management issue though).
1. Others?

Reply:
Per our discussion, I went with option 1